frontend/src/metabase/lib/markdown/parseMarkdown.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Options } from "react-markdown/lib/rehype-filter";
+import rehypeFilter from "react-markdown/lib/rehype-filter";
+import remarkGfm from "remark-gfm";
+import remarkParse from "remark-parse";
+import remarkRehype from "remark-rehype";
+import { unified } from "unified";
+
+const REMARK_PLUGINS = [remarkGfm];
+
+export const parseMarkdown = (value: string, options: Options = {}) => {
```

Comment:
I don't see tests for `markdown` functions

Reply:
> I don't see tests for markdown functions

When I try to test `parseMarkdown` which imports `remark-parse` (or any other `remark-*` lib) I get an error because it uses ESM import syntax.
`react-markdown` luckily comes with `min.js` which is [configured to be imported instead](https://github.com/metabase/metabase/blob/master/jest.unit.conf.json#L8) of default ESM.
`remark-*` libs come as pure ESM and if we import them directly into our code, tests fail.

@ranquild any suggestions for this?

Reply:
> any suggestions for this?

I had to enable transformations for these dependencies: 9236735f1b9c659dadaeff24c1d6136b5f0f9951
These are naturally required for unit tests of `PinnedItemCard` anyway.

Reply:
> I don't see tests for `markdown` functions

d2290e2056803e6f7495e83743b4fb5213bcc541 + fbcc2b9d0817738cbe2ca75649906d6fd2b00e1c